src/saving/SaveList.cs

```diff
@@ -254,7 +260,16 @@ private void OnConfirmDelete()
         }
 
         GD.Print("Deleting save: ", saveToBeDeleted);
-        SaveHelper.DeleteSave(saveToBeDeleted);
+        try
+        {
+            SaveHelper.DeleteSave(saveToBeDeleted);
+        }
+        catch (SaveHelper.FailedToDeleteSaveException e)
+        {
+            errorSaveDeletionFailed.ShowError(TranslationServer.Translate(
+                "Failed to delete save."), e.Message, e.StackTrace);
```

Comment:
Instead, show the exception message (containing the file's full path) in the Godot error print, which can be added below this.

Also, displaying the stack trace won't do much as we as programmers already know where and how the exception occurs and the error won't change at all in all of the game's lifetime. Let alone showing it to the player.

Reply:
Forgot to mention that the exception message (which in this case is a file path), shouldn't also be shown, this is related to the same choices made in #3947.

Reply:
I see some problems with the translation system, so I'll just drop this link here: https://github.com/Revolutionary-Games/Thrive/blob/master/doc/working_with_translations.md

Reply:
Already ahead of you https://github.com/Revolutionary-Games/Thrive/pull/4431#discussion_r1271449289 🙂